mail/views.py

```diff
@@ -237,3 +238,34 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class MailWebhookView(APIView):
+    """
+    View class that handles Mailgun webhooks
+    """
+    permission_classes = (MailGunWebHookPermission, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
+        """
+        POST method handler
+        """
+        event = request.POST.get("event", None)
+        recipient = request.POST.get("recipient", None)
+        error = request.POST.get("error", None)
+        message_headers = request.POST.get("message-headers", None)
+        log_error_on_bounce = request.POST.get("log_error_on_bounce", "")
+        error_msg = (
+            "Webhook event {event} received by Mailgun for recipient {to}: {error}".format(
+                to=recipient,
+                error="{},{}".format(error, message_headers),
```

Comment:
Please change this back to `error` and add a separate argument for `message_headers` in the format string above

Reply:
Done @noisecapella